src/functions-and-arrays.js

```diff
@@ -1,41 +1,162 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(a, b) {
+  if (a > b) {
+    return a;
+  } else {
+    return b;
+  }
+}
 
+maxOfTwoNumbers(8, 9);
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(longestWord) {
+  if (longestWord.length === 0) {
+    return null;
+  }
+  let magicWord = "";
+  let longest = 0;
+  longestWord.forEach(function(element) {
+    if(longest < element.length) {
+      longest = element.length;
+      magicWord = element
+    }
+  });
+  return magicWord;
+}
+
+findLongestWord(words);
 
 
 
 // Iteration #3: Calculate the sum
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
+function sumNumbers(num) {
```

Comment:
Aqui otra alternativa, buenisimo utilizar forEach 👏 
```suggestion
function sumNumbers(arr) {
  let suma = 0;
  for (value of arr) {
    suma += value;
  }
  return suma;
}
```